reubenjohn/MaRS
# Initialize Python Project and GitHub Repository

closes #2
- **Summary**: Set up a new Python project and initialize a repository on GitHub for version control and collaboration.
- **Steps**:
  1. Create a new directory for the project.
  2. Initialize a Python virtual environment.
  3. Create a `requirements.txt` file for Python dependencies.
  4. Initialize a new Git repository.
  5. Create a `.gitignore` file tailored for Python projects.
  6. Commit the initial project structure to Git.
  7. Create a new repository on GitHub and push the initial commit.
- **Acceptance Criteria**:
  - Python project directory is successfully created.
  - Virtual environment is set up and working.
  - Git repository is initialized with appropriate `.gitignore` settings.
  - Project is successfully pushed to GitHub.
- **Test Scenarios**:
  1. Ensure the Python virtual environment activates correctly.
  2. Verify that initial commit appears in the GitHub repository.
